### DISCUSSION TITLE
Help！Hexo g 无法生成文档页面了，求助大佬！

### DISCUSSION THREAD

**Discussion by Orangenazuna** [theme-shoka-x/hexo-theme-shokaX#180] [Q&A] [answered]
我在配置category_map模块后，并且按照对应在/source/_posts下建立了文件夹放入了.md文件。
首先是_config.yml的配置

```
Category & Tag
#default_category: uncategorized
category_map:
  从零开始网站建立: WebsiteBuilding
tag_map:
```

这是/source/_posts的文件夹顺序：
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/130027357/14b7e1d8-af33-4184-884b-4b493b5ce0bc)
以及，md文件的标签代码：

```
---
title: 完成了个人Blog创建!
date: 2023-12-21
comment: false   # true/false对应开启/关闭本文章评论
sticky  : true   # 置顶文章
tags:
- Blog搭建
categories:
    - [从零开始网站建立]
cover: cover.jpg # 文章顶部和文章介绍图（将覆盖文章主页轮播图）
audio:
  - https://music.163.com/song?id=1387098940
audio: false
quiz: false
---
```

有可能是配置文件哪里写错了，现在的网站预览界面没有任何图片，包括头像Icon。
只有index页面没有任何图片，单独访问文章的话是可以的，以下是控制台的错误反馈。
```
Uncaught TypeError: Cannot read properties of null (reading 'addClass')
    at Ni (app.js?v=0.3.9:3:4081)
    at da (app.js?v=0.3.9:9:7657)
    at Aa (app.js?v=0.3.9:9:23983)
```
同时，大佬们可不可以告诉我插件中的lantern，如何修改灯笼显示的文字吗
```
!= theme.footer.lantern.word1
```
这里看不懂~

**Comment by zkz098** [comment]
请提供域名或者 Hexo 完整环境，这可能是一个普遍问题，需要测试。可将环境打包为zip后发送到`zkz20081204#126.com`（把#替换为@）

灯笼文字在配置中修改：
```yaml
footer:
  lantern:
    word1: "xxx"
    word2: "xxx"

```

**Reply by zkz098** [reply, marked as answer]
目前版本 ShokaX 需要至少有一篇非置顶的文章才能正常渲染 index，您提供的环境只有一个置顶文章所以渲染错误了。
已作为 Bug 收录，将在 0.3.10 修复

**Reply by Orangenazuna** [reply]
> 目前版本 ShokaX 需要至少有一篇非置顶的文章才能正常渲染 index，您提供的环境只有一个置顶文章所以渲染错误了。
> 
> 已作为 Bug 收录，将在 0.3.10 修复

明白了！非常感谢！！

**Comment by Orangenazuna** [comment]
> 目前版本 ShokaX 需要至少有一篇非置顶的文章才能正常渲染 index，您提供的环境只有一个置顶文章所以渲染错误了。 已作为 Bug 收录，将在 0.3.10 修复
我又添加了两篇文章，并且将sticky: false   # 非置顶文章设置为了false，为啥依然index和文章页面无法正确显示呀，只有categories页面可以正常渲染，再次麻烦大大了非常抱歉！
